modules/ROOT/pages/aurads/tutorials/arrow-example.adoc

```diff
@@ -1,82 +1,132 @@
 [[connecting-arrow]]
-= Using Apache Arrow
+= Loading and streaming data with Apache Arrow
```

Comment:
```suggestion
= Loading and streaming back data with Apache Arrow
```

The `back` could hint that its the result part

Reply:
Nice, I like it